dipy/nn/cnn_1denoiser/cnn_1denoiser.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.layers import Conv1D, Activation
+from tensorflow.keras.models import *
+from tensorflow.keras.layers import Input
+#from patchify import patchify, unpatchify
+from sklearn.model_selection import train_test_split
+from datetime import datetime
+now = datetime.utcnow().strftime("%Y%m%d%H%M%S")
```

Comment:
This will happen at import time. What's the purpose of this? If we want to implement logging for tf models I think that we should take a more systematic approach. Happy to discuss that more.

Reply:
Should I remove these?